insights/specs/default.py

```diff
@@ -199,12 +199,17 @@ def is_ceph_monitor(broker):
     cobbler_settings = first_file(["/etc/cobbler/settings", "/conf/cobbler/settings"])
     cobbler_modules_conf = first_file(["/etc/cobbler/modules.conf", "/conf/cobbler/modules.conf"])
     corosync = simple_file("/etc/sysconfig/corosync")
-    corosync_cmapctl = foreach_execute(
+    corosync_cmapctl_on_rhel8 = foreach_execute(
         [
             "/usr/sbin/corosync-cmapctl",
             '/usr/sbin/corosync-cmapctl -m stats "stats.schedmiss."',
             '/usr/sbin/corosync-cmapctl -C schedmiss'
-        ], "%s")
+        ], "%s", deps=[IsRhel8])
+    corosync_cmapctl_on_rhel7 = foreach_execute(
+        [
+            "/usr/sbin/corosync-cmapctl",
+            'corosync-cmapctl -d runtime.schedmiss.timestamp && corosync-cmapctl -d runtime.schedmiss.delay',
+        ], "%s", deps=[IsRhel7])
```

Comment:
Better do it in this way:
```suggestion
    @datasource([IsRhel7, IsRhel8])
    def corosync_cmapctl_cmd_list(broker):
        if broker.get(IsRhel7):
            return ["/usr/sbin/corosync-cmapctl",  '/usr/sbin/corosync-cmapctl -m stats "stats.schedmiss.", '/usr/sbin/corosync-cmapctl -C schedmiss']
        if broker.get(IsRhel8):
            return ["/usr/sbin/corosync-cmapctl",  'corosync-cmapctl -d runtime.schedmiss.timestamp", 'corosync-cmapctl -d runtime.schedmiss.delay']
        raise SkipComponent()

    corosync_cmapctl = foreach_execute(corosync_cmapctl_cmd_list, "%s")
```

Reply:
@xiangce, cool, I'll update it.